telemetry/definitions/commonDefinitions.json

```diff
@@ -317,6 +312,11 @@
             "type": "boolean",
             "description": "If dependencies are copied into upload artifact."
         },
+        {
+            "name": "codeTransformErrorMessage",
```

Comment:
yes. please try to reuse types if possible. create your types only if you are exhautsing all existing error types. 
Also I would recommend creating a general type for errors instead of starting with a prefix